search/elastic.py

```diff
@@ -530,14 +509,18 @@ def search(self,
             )
         """
 
-        log.debug("searching index with %s", query_string)  # lint-amnesty, pylint: disable=unicode-format-string
+        log.debug("searching index with %s", query_string)
 
         elastic_queries = []
         elastic_filters = []
 
-        # We have a query string, search all fields for matching text within the "content" node
+        # We have a query string, search all fields for matching text
+        # within the "content" node
         if query_string:
-            query_string = query_string.translate(query_string.maketrans('', '', RESERVED_CHARACTERS))
+            query_string = query_string.translate(
+                query_string.maketrans('', '', RESERVED_CHARACTERS)
```

Comment:
I see that somewhere you use single quotes, and somewhere double. Here you need to be a man and choose one.

Reply:
Done, I choose double